tests/utils.py

```diff
@@ -17,3 +18,20 @@ def check_subproc_run_for_nonzero(completed_proc: subprocess.CompletedProcess) -
     except Exception as exc:
         # raise the exception with decoded output from linkchecker for readability
         raise Exception(completed_proc.stdout.decode()) from exc
+
+
+def run_cli_command(command: str) -> Tuple[str, str, int]:
+    """
+    Run a CLI command using subprocess and capture the output and return code.
+
+    Args:
+        command (list): The command to run as a list of strings.
+
+    Returns:
+        tuple: (str: stdout, str: stderr, int: returncode)
+    """
+
+    result = subprocess.run(
+        command.split(" "), capture_output=True, text=True, check=False
```

Comment:
It's unclear to me where/who `run_cli_command()` runs, but it feels a bit dangerous to split by a space always and by default

Reply:
I have to agree with @gwaybio; there are situations, e.g. quoted strings, where you'd want to not tokenize the string on just spaces. One option is to use [shlex.split](https://docs.python.org/3.13/library/shlex.html#shlex.split), which will split a string into tokens as a shell would.

Reply:
Thanks - I'll go ahead and make a change here to address the uncertainty here.